docs/index.asciidoc

```diff
@@ -23,6 +23,8 @@ include::{include_path}/plugin_header.asciidoc[]
 The mutate filter allows you to perform general mutations on fields. You
 can rename, remove, replace, and modify fields in your events.
 
+Warning: Each mutate must be in its own code block. See bug https://github.com/logstash-plugins/logstash-filter-mutate/issues/27
```

Comment:
Sorry I had no idea there was a an order-of-operations in the first place. By the looks of it that's a restriction that could be hard to overcome. It may also be the case that by documenting the order it is a) stale the moment it changes or is no longer relevant b) more complex to read and understand at-a-glance.

I'm leaning toward keeping my existing patch as-is, mainly because it's blindingly obvious to anyone reading the resultant configuration file what is happening an in what order.